.devcontainer/create_dockerfile_devcontainer.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
```

Comment:
Please add notes on how to use this in the [README](https://github.com/pnnl/ExaGO/blob/viz-mpi-tutorial-dev/docs/devcontainer/README.md) :)

Reply:
Done